x/marker/keeper/marker.go

```diff
@@ -697,6 +700,62 @@ func (k Keeper) TransferCoin(ctx sdk.Context, from, to, admin sdk.AccAddress, am
 	return nil
 }
 
+// IbcTransferCoin transfers restricted coins between to chains when the administrator account holds the transfer
+// access right and the marker type is restricted_coin
+func (k Keeper) IbcTransferCoin(
+	ctx sdk.Context,
+	sourcePort, sourceChannel string,
+	token sdk.Coin,
+	sender, admin sdk.AccAddress,
+	receiver string,
+	timeoutHeight clienttypes.Height,
+	timeoutTimestamp uint64,
+	checkRestrictionsHandler ibckeeper.CheckRestrictionsHandler) error {
+	m, err := k.GetMarkerByDenom(ctx, token.Denom)
+	if err != nil {
+		return fmt.Errorf("marker not found for %s: %w", token.Denom, err)
+	}
+	if m.GetMarkerType() != types.MarkerType_RestrictedCoin {
```

Comment:
Does it matter if the marker is restricted? Can they get around this by using a normal transfer on unrestricted coins? If that is the case, do you think it would convenient if we just encourage users to send all types of markers with this implementation?

Reply:
I don't believe that users should use this api as the endpoint requires transfer auth to be granted and that grant can only be made for restricted coin markers. Adding extra logic to bypass this requirement seems unwarranted when there is an existing and more common api on the bank module already.